src/actions/send.js

```diff
@@ -0,0 +1,89 @@
+import { createActions } from 'redux-actions';
+
+import { showAlert } from '../utils/alerts';
+import { wallet } from '../utils/wallet';
+import { WalletError } from '../utils/walletError';
+
+export const TOKEN_TYPES = {
+    NEAR: 'NEAR',
+    NEP141: 'NEP141'
+};
+
+export const transfer = ({ 
+    type,
+    isStorageBalanceAvailable,
+    params: {
+        contractName, 
+        amount, 
+        receiverId,
+        memo
+    }
+}) => async (dispatch, getState) => {
+    if (type === TOKEN_TYPES.NEAR) {
+        const { transaction, status } = await dispatch(send.transfer.near(receiverId, amount));
+
+        if (status?.SuccessValue) {
+            dispatch(send.setTxStatus(transaction.hash, 'success'));
+        }
+    } else if(type === TOKEN_TYPES.NEP141) {
+        if (isStorageBalanceAvailable === false) {
+            await dispatch(send.payStorageDeposit(contractName, receiverId));
+        }
+
+        const { transaction: { hash }, status } = await dispatch(send.transfer.nep141({
+            token: { 
+                contractName,
+                metadata: { 
+                    decimals: getState().tokens.tokens[contractName].decimals
+                }
+            },
+            amount,
+            receiverId,
+            memo
+        }));
+
+        if (status?.SuccessValue === '') {
+            dispatch(send.setTxStatus({
+                hash,
+                newStatus: 'success'
+            }));
+        }
+    } else {
+        throw new WalletError(`Could not transfer unsupported token: ${type}`, 'send.unsupportedToken', { type });
+    }
+};
+
+export const { send } = createActions({
+    SEND: {
+        TRANSFER: {
+            NEAR: [
+                wallet.sendMoney.bind(wallet),
+                (receiverId, amount) => ({
+                    ...showAlert({ onlyError: true }),
+                    receiverId,
+                    amount
+                })
+            ],
+            NEP141: [
+                wallet.fungibleTokens.transfer?.bind(wallet.fungibleTokens),
+                ({
+                    amount,
+                    receiverId
+                }) => ({
+                    ...showAlert({ onlyError: true }),
```

Comment:
Reducer receives `payload` from `FungibleTokens.transfer` method, that will be the result of tx - this data is not there. We can return it from `FungibleTokens.transfer` method, but passing it as meta seems like a better idea because `transfer` method will return tx result only.